### ISSUE TITLE
Allow the schema to be displayed in IDL format

### ISSUE THREAD

**Issue by bbakerman** [graphql/graphiql#582]
Today we have the documentation view of the schema rendered as HTML

It would be great to have a link that showed the IDL view of the schema or type

eg

     type Person {
          # The name of this person.
          name: String
          # The year were born
          birthYear: String
          eyeColor: String
          gender: String
          hairColor: String
          height: Int
          mass: Float
          skinColor: String
          homeworld: Planet
    }

This would allow you to capture the schema in IDL format as well as given you a way to stare at the schema as a whole versus the in detail view of today

**Comment by asiandrummer** [comment]
@bbakerman it's an interesting idea. I think a link to open a new tab, **convert** the schema into the IDL format, and show the IDL in the new tab would be a good way to, say, export the schema into IDL format from GraphiQL. Would be stoked for a PR for this.

**Comment by flieks** [comment]
UP +1 :). This is also great for migration to another graphql service

**Comment by timshaw9791** [comment]
any arrangement for this issue?

**Comment by imolorhe** [comment]
Looks like a good fit for a plugin in 2.0 @acao

**Comment by acao** [comment]
agreed, @imolorhe lets label “potential plugin” and add it to the roadmap then!
